docs/source/daal/usage/algorithms.rst

```diff
@@ -34,9 +34,12 @@ In computation modes that permit multiple calls to the ``compute()`` method,
 ensure that the structure of the input data, that is, the number of features, their order, and type, is the same for all the calls.
 The following methods are available to provide input to an algorithm:
 
-.. list-table::
+.. tabularcolumns::  |\Y{0.2}|\Y{0.8}|
+
+.. list-table:: Algorithm Input
    :widths: 10 60
    :align: left
+   :class: longtable
 
    * - ``input.set(Input ID, InputData)``
      - Use to set a pointer to the input argument with the ``Input ID`` identifier. 
```

Comment:
There are illegal line breaks in the bulleted items below the list-table on this page: \docs\source\daal\usage\algorithms.rst:

Please correct them. They prevent the LaTeX build. Thanks.